notebooks/official/pytorch/pytorch-text-sentiment-classification-custom-train-deploy.ipynb

```diff
@@ -0,0 +1,1979 @@
+{
```

Comment:
<p>sentiment analysis task -&gt; a sentiment analysis task</p><p> </p><p>the Vertex AI -&gt; Vertex AI</p><p> </p><p>predictions service -&gt; prediction service</p><p> </p><p>Deploying model on Vertex AI Predictions with custom container&nbsp;-&gt; Deploying a model on Vertex AI using a custom container</p><p> </p><p>a HTTP -&gt; an HTTP</p><p> </p><p>container image requirements to be compatible with Vertex AI Predictions. -&gt; prediction container requirements on Vertex AI.</p><p> </p><p>following -&gt; the following</p><p> </p><p>Vertex AI Predictions -&gt; Vertex AI</p><p> </p><p>Torchserve -&gt; TorchServe</p><p> </p><p>custom container image -&gt; the custom container image</p><p> </p><p>Vertex AI Model -&gt; Vertex AI model</p><p> </p><p>Add a line spacing before: Create a custom model handler to handle prediction requests</p><p> </p><p>serving -&gt; serve</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>